### PR TITLE
Add member course

### PR DESCRIPTION
**Pull request by ManhLeDuc**
## Related Tickets
- [#40276](https://edu-redmine.sun-asterisk.vn/issues/40276)

## WHAT (optional)
- Add trainee and supervisor to course

## HOW
- Show list of supervisors, trainees
- Supervisor of course can add supervisors or trainees to course
- Using ajax

## Evidence (Screenshot or Video)
image

image

### CODE REVIEW

**Comment by trannguyen61** [comment]
OK

**Comment by nguyenquanganhc** [comment]
OK

**Comment by ManhLeDuc** [comment]
ready

File: app/controllers/course_members_controller.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  private
+
+  def load_user_by_email
+    @user = User.find_by email: params[:email]
+    return if @user
+
+    fail_respond t("not_found")
+  end
+
+  def success_respond message
```
Comment (ngocvt-0484) [inline app/controllers/course_members_controller.rb]: viết ra application controller để dùng dc ở nhiều chỗ

File: app/views/courses/show.html.erb [hunk 2]
```diff
@@ -15,26 +15,28 @@
...
-    </tbody>
-  </table>
-  <%= paginate @subjects, theme: "bootstrap-5" %>
-  <section class="subject-form center_div">
-    <%= render "subjects/subject_form" %>
-  </section>
+<br>
+<nav>
+  <div class="nav nav-tabs" id="nav-tab" role="tablist">
+    <button class="nav-link active" id="nav-subjects-tab" data-bs-toggle="tab" data-bs-target="#nav-subjects" type="button" role="tab" aria-controls="nav-subjects" aria-selected="true">
```
Comment (ngocvt-0484) [inline app/views/courses/show.html.erb]: dài quá xuống dòng e nhé

File: config/settings.yml [hunk 3]
```diff
@@ -1,5 +1,7 @@
 pic_url: "https://picsum.photos/seed/picsum/200/300"
 date_format: "%d/%m/%Y"
+trainees_body_id: "traineesBodyId"
```
Comment (ngocvt-0484) [inline config/settings.yml]: k cần cho vô setting tên class hay id đâu :((

**Comment by ManhLeDuc** [comment]
ready

**Pull request merged**